node/Cargo.toml

```diff
@@ -24,11 +24,11 @@ nix = "0.23"
 serde = { version = "1.0.101", features = [ "derive" ] }
 serde_json = "1.0"
 tempfile = "3.2.0"
-tracing-core = "=0.1.26"
```

Comment:
Could use review

Reply:
I confirm that this line can indeed be deleted now.